scripts/export.js

```diff
@@ -6,9 +6,27 @@ $(document).ready(() => {
             name_select.append(
                 $('<option></option>').val(text).html(text)
             );
-        })
+        });
     });
 
+    $.get('/all-batch-info', data => {
```

Comment:
I agree we should definitely try to move this to a template (though it's fine for now)!